frontend/src/api/mockAPI/sessions.js

```diff
@@ -26,6 +26,36 @@ export class Session extends MockAPIController {
             throw new Error(message);
         }
     }
+    validateUpdate(session) {
+        if ("name" in session) {
+            if (session.name === undefined || session.name.length === 0) {
```

Comment:
If `session.name` is `undefined`, doesn't `session.name.length` give a TypeError?